common/rpc-models.go

```diff
@@ -148,6 +148,9 @@ type CopyJobPartOrderRequest struct {
 	S2SInvalidMetadataHandleOption InvalidMetadataHandleOption
 	S2SPreserveBlobTags            bool
 	CpkOptions                     CpkOptions
+
+	PrimaryCredentialType   CredentialType
```

Comment:
The naming is a bit confusing, how do these two credential types interact with each other?